<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Background-color :white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCanvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>파일(&amp;F)</string>
    </property>
    <addaction name="action_NEW"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_T">
    <property name="title">
     <string>도구(&amp;T)</string>
    </property>
    <widget class="QMenu" name="menu_P">
     <property name="title">
      <string>펜(&amp;P)</string>
     </property>
     <addaction name="actionRED_Red"/>
     <addaction name="actionGreen_Green"/>
     <addaction name="actionBLUE_Blue"/>
    </widget>
    <widget class="QMenu" name="menuchang_T">
     <property name="title">
      <string>변환(&amp;T)</string>
     </property>
     <addaction name="action_G"/>
    </widget>
    <addaction name="menu_P"/>
    <addaction name="menuchang_T"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>도움말(&amp;H)</string>
    </property>
    <addaction name="action_action"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_T"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_NEW"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="actionRED_Red"/>
   <addaction name="actionGreen_Green"/>
   <addaction name="actionBLUE_Blue"/>
   <addaction name="separator"/>
   <addaction name="action_action"/>
   <addaction name="action_Exit"/>
  </widget>
  <action name="action_NEW">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/new.png</normaloff>:/icon/imgs/new.png</iconset>
   </property>
   <property name="text">
    <string>새 문서(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/open.png</normaloff>:/icon/imgs/open.png</iconset>
   </property>
   <property name="text">
    <string>열기(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/save.png</normaloff>:/icon/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>저장(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/exit.png</normaloff>:/icon/imgs/exit.png</iconset>
   </property>
   <property name="text">
    <string>종료(&amp;X)</string>
   </property>
  </action>
  <action name="actionRED_Red">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/red.png</normaloff>:/icon/imgs/red.png</iconset>
   </property>
   <property name="text">
    <string>RED(&amp;R)</string>
   </property>
  </action>
  <action name="actionGreen_Green">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/green.png</normaloff>:/icon/imgs/green.png</iconset>
   </property>
   <property name="text">
    <string>Green(&amp;G)</string>
   </property>
  </action>
  <action name="actionBLUE_Blue">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/blue.png</normaloff>:/icon/imgs/blue.png</iconset>
   </property>
   <property name="text">
    <string>BLUE(&amp;B)</string>
   </property>
  </action>
  <action name="action_action">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icon/imgs/about.png</normaloff>:/icon/imgs/about.png</iconset>
   </property>
   <property name="text">
    <string>이 프로그램은...(&amp;a)</string>
   </property>
  </action>
  <action name="action_G">
   <property name="text">
    <string>그레이스케일(&amp;G)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
